docker/docker-compose.yml

```diff
@@ -1,61 +1,86 @@
-logstash:
-  extends:
-    file: common.yml
-    service: logstash
-  links:
-  - elastic
+version: "2.4"
+services:
 
-mongodb:
-  extends:
-    file: common.yml
-    service: mongodb
+  mongodb:
+    image: mongo:3
+    container_name: mongodb
 
-redis:
-  extends:
-    file: common.yml
-    service: redis
+  redis:
+    image: redis:3
+    container_name: redis
 
-kibana:
-  extends:
-    file: common.yml
-    service: kibana
-  links:
-  - elastic
-  ports:
-  - "5601:5601"
+  elastic:
+    image: elasticsearch:7.7.1
+    container_name: elastic
+    environment:
+    - xpack.security.enabled=false
+    - discovery.type=single-node
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+    cap_add:
+      - IPC_LOCK
 
-elastic:
-  extends:
-    file: common.yml
-    service: elastic
+  postfix:
+    image: catatnight/postfix
+    container_name: postfix
+    environment:
+    - maildomain=mail.sourcefabric.org
+    - smtp_user=user:pwd
 
-postfix:
-  extends:
-    file: common.yml
-    service: postfix
+  superdesk-server:
+    build: ../server/
+    container_name: superdesk-server
+    links:
```

Comment:
if working with docker-file 3.x links are not necessary anymore (we just need to set `depents_on`, and the default network setup should be fine).